contracts/optimistic-ethereum/OVM/bridge/tokens/OVM_L1ETHGateway.sol

```diff
@@ -48,6 +48,13 @@ contract OVM_L1ETHGateway is iOVM_L1ETHGateway, OVM_CrossDomainEnabled, Lib_Addr
      * Depositing *
      **************/
 
+    receive()
```

Comment:
Is `receive` a new thing?

Reply:
I see https://docs.soliditylang.org/en/latest/060-breaking-changes.html#semantic-and-syntactic-changes